_data/banking.yml

```diff
@@ -715,6 +715,15 @@ websites:
       facebook: nubankbrasil
       lang: pt
 
+    - name: Openbank
+      url: https://www.openbank.es
+      img: openbank.png
+      tfa: No
+      twitter: openbank_es
+      facebook: openbank.es
+      lang: es
+      lang: es
```

Comment:
```suggestion
```

Reply:
Resolved by reverting the addition of the double `lang`.